pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -312,6 +311,27 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 	return managedresources.CreateForSeed(ctx, b.client, b.namespace, managedResourceControlName, false, resources)
 }
 
+func getDruidDeployCommands(config *config.GardenletConfiguration) []string {
+	command := []string{"" + "/bin/etcd-druid"}
+	command = append(command, "--enable-leader-election=true")
+	command = append(command, "--ignore-operation-annotation=false")
+	command = append(command, "--workers=50")
+	command = append(command, "--disable-lease-cache=true")
```

Comment:
```suggestion

```
We already watch `Lease` objects for backup compaction, so there is no point of disabling the cache here. We shall remove the flag in the upcoming Druid release.

Reply:
Okay. Done